### PR TITLE
test:Created ./keadm/common/step_test.go

### PR DESCRIPTION
**Pull request by vivekbisen04**
/kind test  
/kind coverage  

## What this PR does / why we need it

This PR significantly improves test coverage for the `keadm/cmd/keadm/app/cmd/common` package. It ensures comprehensive test coverage by verifying:

- Command execution workflows
- Argument validation logic
- Coverage of edge cases

## Which issue(s) this PR fixes

Part of [lfx-mentorship] Enhance KubeEdge testing coverage initiative #6101

## Screenshot
Screenshot from 2025-02-06 19-35-00

## Command to test

- **File path:** `keadm/cmd/keadm/app/cmd/common/step_test.go`
- **Test execution command:**

  ```sh
  go test -coverprofile=coverage.out ./keadm/cmd/keadm/app/cmd/common/...
  go tool cover -html=coverage.out -o coverage.html

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/common/step_test.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The KubeEdge Authors.
```
Comment (fujitatomoya) [inline keadm/cmd/keadm/app/cmd/common/step_test.go:2]: since this is newly added, copyright year should be adjusted for current year.

```suggestion
Copyright 2025 The KubeEdge Authors.
```

**Review by fujitatomoya — APPROVED** [review]
lgtm with copyright update.

**Review by wbc6080 — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
/approve
